shelley/chain-and-ledger/executable-spec/src/EpochBoundary.hs

```diff
@@ -190,6 +190,7 @@ groupByPool active delegs =
     | hk <- Map.keys delegs
     ]
 
+-- | Snapshot of the stake pools, and pool parameters.
```

Comment:
it's really a snapshot of the stake distribution

Reply:
Fixed.